modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+The users of a project are assigned to a cluster role based on their access control. These cluster roles are mentioned in the *Project* view > *Project access* tab > *Role* drop down. By default, these roles are *Admin*, *Edit*, and *View*. To add or edit the cluster roles for a project, you can customize the YAML code of the cluster.
+
+.Project roles
+image::odc_cluster_role.png[]
+
```

Comment:
Suggest dropping this screenshot from maintenance pov. The instructions are clear enough. Also, the previous module has a similar screenshot, though that might not be relevant when we consider each module to be self sufficient. But like I said the instructions are clear in this case.

Reply:
Agreed with your point on this. Let me remove the screenshot from here.